### ISSUE TITLE
[10.0.0] Stats graphs are not working anymore

### ISSUE THREAD

**Issue by stumbaumr** [getsentry/onpremise#379]
I am running 10.0.0
```
root@sentry01(dc1.prd):~# docker ps
CONTAINER ID        IMAGE                                  COMMAND                  CREATED             STATUS                  PORTS                          NAMES
efe392a1acf5        sentry-onpremise-local                 "/bin/sh -c 'exec $S…"   2 hours ago         Up 2 hours              9000/tcp                       sentry_onpremise_worker_1
5982925a4fc8        sentry-onpremise-local                 "/bin/sh -c 'exec $S…"   2 hours ago         Up 2 hours              0.0.0.0:9000->9000/tcp         sentry_onpremise_web_1
7e01f4391ab4        sentry-onpremise-local                 "/bin/sh -c 'exec $S…"   2 hours ago         Up 2 hours              9000/tcp                       sentry_onpremise_post-process-forwarder_1
b1087933ef4e        sentry-cleanup-onpremise-local         "/entrypoint.sh '0 0…"   2 hours ago         Up 2 hours              9000/tcp                       sentry_onpremise_sentry-cleanup_1
ffbdad7467ad        sentry-onpremise-local                 "/bin/sh -c 'exec $S…"   2 hours ago         Up 2 hours              9000/tcp                       sentry_onpremise_cron_1
65d3fe862aa6        getsentry/snuba:latest                 "./docker_entrypoint…"   2 hours ago         Up 2 hours              1218/tcp                       sentry_onpremise_snuba-api_1
a9e707455ecb        snuba-cleanup-onpremise-local          "/entrypoint.sh '*/5…"   2 hours ago         Up 2 hours              1218/tcp                       sentry_onpremise_snuba-cleanup_1
9293b1186c72        getsentry/snuba:latest                 "./docker_entrypoint…"   2 hours ago         Up 2 hours              1218/tcp                       sentry_onpremise_snuba-replacer_1
535c3d6f18aa        getsentry/snuba:latest                 "./docker_entrypoint…"   2 hours ago         Up Less than a second   1218/tcp                       sentry_onpremise_snuba-consumer_1
3b26c4697aa4        confluentinc/cp-kafka:5.1.2            "/etc/confluent/dock…"   2 hours ago         Up 2 hours              9092/tcp                       sentry_onpremise_kafka_1
932b9c176bbb        getsentry/symbolicator:latest          "/bin/bash /docker-e…"   2 hours ago         Up 2 hours              3021/tcp                       sentry_onpremise_symbolicator_1
f48add1a160b        postgres:9.6                           "docker-entrypoint.s…"   2 hours ago         Up 2 hours              5432/tcp                       sentry_onpremise_postgres_1
c119e9ab0515        tianon/exim4                           "docker-entrypoint.s…"   2 hours ago         Up 2 hours              25/tcp                         sentry_onpremise_smtp_1
e120f3c106d6        confluentinc/cp-zookeeper:5.1.2        "/etc/confluent/dock…"   2 hours ago         Up 2 hours              2181/tcp, 2888/tcp, 3888/tcp   sentry_onpremise_zookeeper_1
388f6992cb18        yandex/clickhouse-server:19.4          "/entrypoint.sh"         2 hours ago         Up 2 hours              8123/tcp, 9000/tcp, 9009/tcp   sentry_onpremise_clickhouse_1
2fa3b131ad24        symbolicator-cleanup-onpremise-local   "/entrypoint.sh '55 …"   2 hours ago         Up 2 hours              3021/tcp                       sentry_onpremise_symbolicator-cleanup_1
f5779efa7859        memcached:1.5-alpine                   "docker-entrypoint.s…"   2 hours ago         Up 2 hours              11211/tcp                      sentry_onpremise_memcached_1
7a5e77c62d2a        redis:5.0-alpine                       "docker-entrypoint.s…"   2 hours ago         Up 2 hours              6379/tcp                       sentry_onpremise_redis_1
root@sentry01(dc1.prd):~#
```
As you can see in the below screenshot there was an exception reported, but since three days there is no update with the stats graph anymore.
 
![grafik](https://user-images.githubusercontent.com/12425441/73782676-c8548100-4792-11ea-9644-a2cc07b6ff7a.png)

**Comment by BYK** [comment]
This usually indicates an issue with Redis or the workers. Can you check their logs and see if anything is wrong?

**Comment by stumbaumr** [comment]
So I fixed it. Took some time.

I upgraded sentry and that worked out of the box. Kudos for that!

But then I had to add some parameters and had to rebuild the container. When running docker-compose up I got newer versions of snuba and symbolicator.

After fiddling around for some time I did a comparison of the current state of the master and found commits https://github.com/getsentry/onpremise/commit/37d6166032c6acb1601b9ee4f7b2b3ddff0db7eb and https://github.com/getsentry/onpremise/commit/fd9bd107df57637f6901e14716ea62a63ba4cd3b and applied them.

After the next docker-compose up it was working again...
